### PR TITLE
Matt/update homepage small changes1

### PR DESCRIPTION
**Pull request by mattdias**
some small adjustment to sizes and spaces on the homepage
removed options for announcements

### CODE REVIEW

File: app/css/template.css [hunk 1]
```diff
@@ -24,7 +24,9 @@ div.km-pre {
 .bold {
     font-weight: bold!important;
 }
-
+html {
+    scroll-behavior: smooth;
```
Comment (blaisefonseca) [inline app/css/template.css:28]: please not global behaviour, why is this needed?

File: app/views/home/bch.html [hunk 2]
```diff
@@ -42,12 +42,13 @@
                     {{::('bchHomeT.announcements'|$translate)}}
                     <hr class="m-0 color-white">
                 </p>
-                <div ng-if="announcementCounts" class="pt-3">
-                    <home-articles tags="bch-announcement" layout="vertical" show-summary="true" counts="{{announcementCounts}}"></home-articles>
+                <div class="pt-3">
```
Comment (blaisefonseca) [inline app/views/home/bch.html:45]: `ng-if="announcementCounts"` this should stay

File: app/css/template.css (hunk 1 above)
Comment (mattdias) [reply app/css/template.css]: removed

**Review by blaisefonseca — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]
